app/models/agents/http_status_agent.rb

```diff
@@ -66,7 +66,8 @@ def check
     def receive(incoming_events)
       incoming_events.each do |event|
         interpolate_with(event) do
-          check_this_url interpolated[:url], header_array(interpolated[:headers_to_save])
+          check_this_url event.payload[:url] || interpolated[:url],
```

Comment:
Do we even need `event.payload[:url]`? Even if it seems intended in the specs, favoring the "raw" value could change the behavior of existing agents, say having `{{ url | replace 'something', 'else' }}` as the url option.